src/styles/main.scss

```
@@ -1,3 +1,11 @@
-body {
-  margin: 0;
-}
+@import "./utils/reset.scss";
+@import "./utils/normalize.scss";
+@import "./utils/variables.scss";
+@import "./utils/functions.scss";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/card.scss";
+@import "./blocks/reviewSection.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/price.scss";
+@import "./blocks/container.scss";
```

Empty line between utils and blocks in here
```suggestion
@import "./utils/reset.scss";
@import "./utils/normalize.scss";
@import "./utils/variables.scss";
@import "./utils/functions.scss";

@import "./blocks/header.scss";
@import "./blocks/nav.scss";
@import "./blocks/card.scss";
@import "./blocks/reviewSection.scss";
@import "./blocks/stars.scss";
@import "./blocks/price.scss";
@import "./blocks/container.scss";
```